internal/app/db/sessions.go

```diff
@@ -1,11 +1,18 @@
 package db
 
-import "go.mongodb.org/mongo-driver/mongo"
+import (
+	"context"
+	"go.mongodb.org/mongo-driver/mongo"
+)
 
 type Session struct {
 	Client *mongo.Client
 }
 
+func (sess Session) Close(ctx context.Context) {
+	sess.Client.Disconnect(ctx)
+}
+
 func (sess Session) Users() *mongo.Collection {
 	return sess.Client.Database("entities").Collection("users")
```

Comment:
can avoid hardcoding these right

Reply:
I guess the database name can be an environment variable, but the collection name can be a string literal since these functions are just like constants